src/elements/select/select.svelte

```diff
@@ -105,12 +106,14 @@ const handleEnter = () => {
 
     if (result) {
       value = result;
+      dispatch('select', { value });
```

Comment:
To match [vanilla `<select>`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement#events), should this event be `change` instead of `select`?

> When dealing with a vanilla `<select>`, the [following events](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement#events) can be emitted:
>
> - [change](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event) event
> Fires when the user selects an option.
> - [input](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event) event
> Fires when the value of an [<input>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input), [<select>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select), or [<textarea>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) element has been changed.

Reply:
that makes sense to match the behavior, will change naming to "change"